apps/web/vibes/soul/primitives/products-carousel/index.tsx

```diff
@@ -27,13 +27,10 @@ export function ProductsCarousel({
   return (
     <Carousel className={className}>
       <CarouselContent className="mb-10">
-        <Stream
-          value={streamableProducts}
-          fallback={<ProductsCarouselSkeleton className={className} message={emptyStateMessage} />}
-        >
+        <Stream value={streamableProducts} fallback={<ProductsCarouselSkeleton />}>
```

Comment:
I think the reason we were passing the class was to control with in Makeswift. Might need to rethink later by wrapping in div and passing class there

Reply:
It is weird that the `className` is being passed to the `Carousel` component but that same to the Skeleton (which is nested within `CarouselContent`). Seems like an oversight to me?

Reply:
That could be a mistake